modules/gateway/peers.go

```diff
@@ -26,10 +24,37 @@ const (
 	minNodeListLen = 100
 )
 
+var (
+	// The gateway will sleep this long between incoming connections.
+	acceptInterval = func() time.Duration {
```

Comment:
that actually could be a separate attack vector. We aren't operating at the same sensitivity as Bitcoin, but for Bitcoin a 3 second delay for miners is a really big deal, it puts them 0.5% up in orphan rate compared to someone who is skipping the 3 second delay.

This is fine for now, but overall we should definitely have a different method for handing DoS in the gateway.